SSPSolution/SSPSolution/System.cpp

```diff
@@ -511,20 +511,20 @@ void System::CreateDummyObjects()
 	tempPos = DirectX::XMVectorSet(0, 1.0, 60, 0);
 	ptr = this->m_physicsHandler.CreatePhysicsComponent(tempPos, false);
 	ptr->PC_mass = 200;
-	ptr->PC_AABB.ext[0] = 1;
-	ptr->PC_AABB.ext[1] = 1;
-	ptr->PC_AABB.ext[2] = 1;
+	ptr->PC_AABB.ext[0] = 0.3;
+	ptr->PC_AABB.ext[1] = 0.3;
+	ptr->PC_AABB.ext[2] = 0.3;
```

Comment:
Please add the "f" as the postfix to the value: 
0.3f

Same goes for line 514-516,
523-525 were it should be 0.5f,
and finally 0.2f at line 527.
It should be readable that we're handling floats.